packages/esm-ward-app/src/ward-workspace/patient-transfer-bed-swap/patient-transfer-request-form.component.tsx

```diff
@@ -102,7 +102,7 @@ export default function PatientTransferForm({
       }
 
       createEncounter({
-        patient: patient.uuid,
+        patient: patient?.uuid,
```

Comment:
What is the underlying issue here?  You should never be able to create an encounter without a patient, so if patient is null here there is a bigger problem?

Reply:
Right, so I have added that the save button is only enabled after the patient is loaded.

Reply:
Thanks @usamaidrsk .... what I was wondering was the "patient" variable would ever be null prior to "createEncounter" function being *called*.  But I think I understand now... since createEncounter is within a function that is being defined using "useCallback", it's actually trying to resolve patient.uuid *prior* to createEncounter actually being called?